skiko/src/iosMain/kotlin/org/jetbrains/skiko/redrawer/MetalRedrawer.ios.kt

```diff
@@ -16,12 +16,15 @@ import platform.Metal.MTLCreateSystemDefaultDevice
 import platform.Metal.MTLDeviceProtocol
 import platform.Metal.MTLPixelFormatBGRA8Unorm
 import platform.QuartzCore.*
-import platform.UIKit.UIScreen
-import platform.UIKit.UIView
 import platform.UIKit.window
-import platform.darwin.NSInteger
 import platform.darwin.NSObject
 
+internal enum class DrawSchedulingState(val canDrawNow: Boolean, val hasDrawScheduledForNextFrame: Boolean) {
```

Comment:
canDrawNow is not accessed,  hasDrawScheduledForNextFrame can be replaced with == SCHEDULED_ON_NEXT_FRAME 
So I  believe this enum properties can be removed

Reply:
Oh, it's an old commit, it was already fixed.

Reply:
Noticed it at the same time, fixed